tests/Nelmio/Alice/support/fixtures/reference_range_2.yml

```diff
@@ -0,0 +1,9 @@
+Nelmio\Alice\support\models\Group:
+    group_1_{@user1}:
+        name: <company()>
+        members: '@user{1..3}'
```

Comment:
nope, this is already a part.

see https://github.com/nelmio/alice/blob/master/doc/relations-handling.md